src/ServiceFabric.Mocks/MissingActorEventArgs.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Microsoft.ServiceFabric.Actors;
+
+namespace ServiceFabric.Mocks
+{
+          
+
+    public class MisingActorEventArgs : EventArgs
```

Comment:
I like the idea.  I considered trying to use reflection to make this all work but the type restrictions are kind of inconsistent in the ServiceFabric libraries.  Sometimes it is requiring generic arguments that are inheriting from Actor, ActorBase, or implementations of IActor.  I am sure you noticed when putting this all together.  Let me try to work on that and see if I can make it work.